test_models.py

```
@@ -0,0 +1,361 @@
+        
+        if loaded_device.configuration_snapshots:
+            snapshot = loaded_device.configuration_snapshots[0]
+            print(f"   ✅ Config->Device relationship: {snapshot.device.hostname}")
+
+
+async def test_crud_operations():
+    """Test basic CRUD operations."""
+    print("🧪 Testing CRUD operations...")
+    
+    async with get_async_session() as session:
+        from sqlalchemy import select, update, delete
```

Ruff (0.12.2)

`sqlalchemy.delete` imported but unused

Remove unused import: `sqlalchemy.delete`

(F401)